LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
```

Comment:
The origin of this was presenting the lock into a document provider extension. On the iPad, the document picker is presented into a modal in the center of the screen (see this: https://mobileonly.tools/img/2016-10-06-Document-Provider-0.PNG)

The locker, then, was uncentered, and there is no way to center it since the window parameters are nil (that's why we need to send the superview)